src/components/Searchnav/Searchnav.jsx

```diff
@@ -1,5 +1,7 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 import Africa from '../../asserts/images/Africa.png';
+import Buttons from '../Buttons/Buttons';
 import './Searchnav.css';
 
 export default function Searchnav() {
```

Comment:
This component is call Searchnav. 

- Firstly it is not in camel case
- Also don't you think it should be named something else. Like is it really a "Search Nav bar"?